beta/index.md

```diff
@@ -0,0 +1,127 @@
+sudo apt-get update
+sudo apt-get install gravwell
+```
+
+### Docker
+
+The Docker images is available at [gravwell/beta](https://hub.docker.com/r/gravwell/beta). You can substitue `gravwell/gravwell` with `gravwell/beta` in any of the docker documentation and it should "just work."
+
+
+## Major features included in Gravwell Big Bang
+
+There are many new features in both the GUI and under the hood, but we would like to focus the beta on the features that may need some work.
+
+### Kits
+
+This is it. The Big Bang, the start, a beginning of time. The supermassive featureset that starts the future of Gravwell.
+
+Kits are pre-packaged use case bundles made of searches, dashboards, resources, and more. Read on to see what kind of awesome stuff can be included.
+
+#### Available Kits
+
+Currently, there are two kits available in the Kit repository. We have a Network Enrichment kit, which provides a bunch of great resources for resolving things like port->service and IP geolocation. We also have a Netflow v5 kit, which comes with overview dashboards, investigative dashboards, query library searches, templates, and a playbook to get you started analyzing netflow data.
+
+NOTE: The current netflow kit REQUIRES that netflow data be coming in under the `netflow` tag. In the future, you will have the option of specifying the tag when you install a kit (since tags are arbitrary). For now, make sure that your netflow data is coming in under the `netflow` tag for your beta Gravwell instance.
+
+See the [netflow ingester docs](https://docs.gravwell.io/docs/#!ingesters/ingesters.md#Netflow_Ingester) for more information.
+
+
+### Query Library
+
+This actually isn't new to the Big Bang release but we feel it deserves another mention due to Kits bundling up saved queries. The query library enables users to save commonly used queries and share with other members of their group. You can now filter based on labels or search using basic text search.
+
+### Playbooks
+
+Playbooks are an opportunity to include READMEs, documentation, directions for activities like threat hunting, context and knowledge about specific internal systems, application log explanations -- YOU NAME IT.
+
+Built using markdown syntax, playbooks can include searches directly for a reader to launch from the playbook. We are actively developing this feature to include more syntactical goodies and are excited about the potential.
+
+### SOAR Scripts
+
+Also not a new feature, but we have expanded the debugging capability and enhanced the UX developing and managing SOAR scripts.
+
+### Actionables
+
+Actionables allow you to set triggers which will make matching text "clickable" when it shows up in the text-based renderers of Gravwell. This can allow you to click an IP address for further investigation, open an AWS security group in the AWS console, or lookup what Shodan knows about a given host. Actionables are also usable when interesting text is highlighted.
+
```

Comment:
it would be awesome to see an animated gif of actionables working here.

Reply:
Not right now.